### PR TITLE
feat:linked pages

### PR DESCRIPTION
**Pull request by Sire6715**
linked the pages using for loops and made the navbar heading display page title for every page.

### CODE REVIEW

File: Copyright.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+layout: policiesLayout
+title: copyright infringement
+permalink: /copyright/
```
Comment (sammy-code98) [inline Copyright.md:4]: great work uche.

i think the extra forward slash '/' in front should be removed

**Review by sammy-code98 — APPROVED** [review]

**Review by copy-ninja1 — CHANGES_REQUESTED** [review]
rename accountTerms to just terms and put it in a diffrent folder (terms/index.md)
put summary.md in it's own folder (summary/index.md) the same thing for other .md files

example
Screenshot from 2023-01-30 12-51-37

**Comment by Sire6715** [comment]
just finished making the changes to the url and the directories.
changed the default on load page to the "copyright infringement" page since its the first on the navigation menu.Awaiting your feedback/review, thank you.

**Review by sammy-code98 — APPROVED** [review]
Great work @Sire6715

**Review by copy-ninja1 — APPROVED** [review]

**Pull request merged**